src/Routes/web.php

```diff
@@ -11,7 +11,10 @@
     ->middleware(['web', HandleInertiaRequests::class])
     ->group(function () {
         Route::get('/', [InertiaController::class, 'home']);
-        Route::get('/course/{id}/people', [InertiaController::class, 'people']);
         Route::get('/register', [InertiaController::class, 'register']);
         Route::get('/login', [InertiaController::class, 'login']);
+
+        Route::prefix('/course')->as('course.')->group(function () {
+            Route::get('/course/{id}/people', [InertiaController::class, 'people']);
```

Comment:
`Route::get('/{id}/people', ... `
`/course` is already defined in the prefix

Reply:
please fix this asap